### ISSUE TITLE
[Design] As a grantee user I want to receive email notifications about the status of my account and data

### ISSUE THREAD

**Issue by reitermb** [raft-tech/TANF-app#958]
Problem Statement:
To enable email comms we need HTML templates _to_ email

Tasks
- [ ] Write content of emails (for notifications that aren't already written)
   - [ ] Confirmed Submission
   - [ ] Access Request Submitted/Approved
   - [ ] Only other notifications pertinent to release 2
- [ ] Design HTML email templates for content

Notes
- Potentially blocked by dev spike re: delivery system for TDP email notifications 

DD
- [ ] Documentation: Design has been shared with the PO and team for feedback
- [ ] @lfrohlich has reviewed and signed off

**Comment by ADPennington** [comment]
agree with adding this to the new ticket @valcollignon @sreedevip @lfrohlich

**Comment by reitermb** [comment]
Now closeable with the addition of #1462 @lfrohlich

**Comment by stevenino** [comment]
@lfrohlich Closing this ticket since it has now been superseded by 1462, 1708 and 1709.

cc: @reitermb @sreedevip
